README.md

```diff
@@ -148,4 +148,9 @@ Many thanks to:
 
 ## Tests
 
-Run unit tests by `python -m unittest tests/test_configs.py` or suites of tests by running `everyvoice test dev` if you have the package installed interactively.
+There are many ways to run the unit tests, if you installed EveryVoice from source:
+ - Run all the dev tests: `everyvoice/run_tests.py dev` or `everyvoice test dev`
```

Comment:
don't you mean `python everyvoice/run_tests.py dev` ?

Reply:
Nope, `run_tests.py` has the shebang line needed to make it directly executable. It works on my machine, and it should work cleanly on any *nix-based OS. The only case where it's not reliable is on the Windows command line. So yes, I mean `everyvoice/run_tests.py dev`.